yt_dlp/extractor/ximalaya.py

```diff
@@ -82,6 +232,24 @@ def _real_extract(self, url):
             audio_info_file, audio_id,
             f'Downloading info json {audio_info_file}', 'Unable to download info file')
 
+        # NOTE(xcsong): VIP-restricted audio
+        if audio_info.get('is_paid', False):
```

Comment:
The default value of get is `None` which is also falsy
```suggestion
        if audio_info.get('is_paid'):
```